tests/integration/tests/poi.js

```diff
@@ -165,9 +165,53 @@ test('display details about the poi on a poi click', async () => {
   expect(wiki_block).not.toBeFalsy()
 })
 
+
+
+test('check poi name behaviour', async () => {
+  expect.assertions(2)
+  let {...poiNoName} = poiMock
+  /* default test with matching name & local_name */
+  poiNoName.id = 'osm:way:453202'
+  nock(/idunn_test\.test/)
+    .persist(false)
+    .get(/osm:way:453202/)
+    .reply(200, JSON.stringify(poiNoName))
+
+  await page.goto(`${APP_URL}/place/osm:way:453202@Musée_dOrsay#map=17.49/2.3261037/48.8605833`)
+
+  await page.waitForSelector('.poi_panel__title')
+  let poiName = await page.evaluate(() => {
+    return  document.querySelector('.poi_panel__title').innerText.trim()
+  })
+
+  expect(poiName).toMatch('Musée d\'Orsay')
+
+  /* set mismatching local_name */
+  poiNoName.local_name = 'Orsay museum'
+  poiNoName.id = 'osm:way:453203'
+  nock(/idunn_test\.test/)
+    .persist(false)
+    .get(/osm:way:453203/)
+    .reply(200, JSON.stringify(poiNoName))
+
+  await page.goto(`${APP_URL}/place/osm:way:453203@Musée_dOrsay#map=17.49/2.3261037/48.8605833`)
+
+  await page.waitForSelector('.poi_panel__title')
+  poiName = await page.evaluate(() => {
+    return  document.querySelector('.poi_panel__title').innerText.trim()
+  })
+
+  expect(poiName).toMatch('Musée d\'Orsay  Orsay museum')
```

Comment:
this looks unreliable. Why two spaces ?  
We should be able to fetch each label independently.